dialogue-target/src/main/java/com/palantir/dialogue/Response.java

```diff
@@ -26,6 +28,9 @@
     /** The HTTP response code for this response. */
     int code();
 
+    /** The HTTP headers for this response. */
```

Comment:
Can we specify that this map uses a case-insensitive comparison?
https://tools.ietf.org/html/rfc7540#section-8.1.2

Reply:
done